docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,199 @@
+    -   **S3BlockUpload** emits **(path, ETag)** to **s3FileMerger**.
+
+-   ***S3FileMergerOperator***
+    -   Complete multipart upload request using **AmazonS3Client.completeMultipartUpload(...)** and must include the **upload ID**, a list of both part numbers and corresponding **ETag** values. **S3FileMergerOperator** send the complete multi-part upload request to S3 once it have all the **part ETag's** of a file. 
+**Amazon S3** creates an object by concatenating the parts in ascending order based on part number. After a successful upload request, the parts no longer exist and S3 response includes an **ETag** which uniquely identifies the combined object data. 
+
+### Configuration Parameters
+-----------------------------
+
+-   ***accessKey*** -   String
+    -   Mandatory Parameter    
+    -   Specifies the AWS access key to access Amazon S3 and have permissions to access the specified bucket.
+
+-   ***secretAccessKey***   -   String
+    -   Mandatory Parameter
+    -   Specifies the AWS secret access key to access Amazon S3 and have permissions to access the specified bucket.
+    
+-   ***endPoint***  -   String
+    -   Endpoint is the url that is entry point for a web service. Specify the valid endpoint to access S3 bucket.
+
+-   ***bucketName***    -   String
+    -   Mandatory Parameter
+    -   S3 buckets are used to store objects which consists of data and metadata that describes the data. Specify the name of the bucket.
+
+-   ***outptuDirectoryPath***   -   String
+    -   Mandatory Parameter
+    -   Specifies the path of the output directory. 
+
+-   ***mergerCount***   -   int
+    -   Specify the number of instances of S3FileMerger operator.
+    -   Default value = 1
+    
+-   ***timeOutWIndowCount***    -   int
+    -   Specify the count as streaming windows. This value will set to the operators in this module because the operators in this module is mostly interacts with the Amazon S3. Due to this reason, window id of these operators might be lag behind with the upstream operators.
+    -   Default value = 6000
+
+### Ports
+---------
+
+-   ***filesMetadataInput***    -   AbstractFileSplitter.FileMetadata
+    -   Input port for files metadata.
+    -   Mandatory
+
+-   ***blocksMetadataInput***   -   BlockMetadata.FileBlockMetadata
+    -   Input port for blocks metadata.
+    -   Mandatory
+    
+-   ***blockData*** -   AbstractBlockReader.ReaderRecord<Slice>
+    -   Input port for blocks data.
+    -   Mandatory
+    
+### Application Example
+-----------------------
+
+Please refer [Example](https://github.com/DataTorrent/examples/tree/master/tutorials/s3output) for S3OutputModule sample application.
+
+### Partitioning
+----------------
+
+Partitioning the module means that operator/operators in the module can able to be partitioned.
```

Comment:
operator/operators in the module can able to be partitioned. =>
the operators in the module can be partitioned.